App.java

```diff
@@ -0,0 +1,89 @@
+
+//import java.io.Console;
+import java.io.*;
+
+public class App {
+    public static void main(String[] args) throws Exception {
+
+        System.out.println("Starting Program!");
```

Comment:
break out this read file function into it's own method that you call from main, and you can pass the file path into it, but just have it return a List<String> so no counting logic etc